### ISSUE TITLE
Giotto suite bug of dimPlot2D

### ISSUE THREAD

**Issue by cranetht** [giotto-suite/Giotto#274]
I observed that when using group_by slot in the dimPlot2D() function, the ggplot2 plot that comes out is completely blank. 

I then checked the source code and found a possible error in https://github.com/RubD/Giotto/blob/suite/R/spatial_visuals.R#L1273 

**subset_cell_IDs = comb_metadata[['cell_ID']][get(group_by) == group]**  may be correct form.

 [['cell_ID']] should be put in front of [get(group_by) == group], not behind it.

**Comment by RubD** [comment]
Thanks @cranetht for letting us know. That should be an easy fix and we'll take a look soon.

**Comment by RubD** [comment]
Hi @cranetht , apologies for the late reply. I took a look at the code, and although I did not change anything, it produces the expected results.  I will close this issue, but if you still have problems and can provide us with a reproducible example (small script), then I'm happy to re-open it.

![Screen Shot 2022-11-19 at 11 15 18](https://user-images.githubusercontent.com/8243389/202860717-9391820a-e49f-458f-a3e3-e79b9a4819f6.png)

![Screen Shot 2022-11-19 at 11 15 04](https://user-images.githubusercontent.com/8243389/202860721-a259cb14-14e6-485d-9530-716e3532e28d.png)
